### ISSUE TITLE
Curiosities 

### ISSUE THREAD

**Issue by ShalokShalom** [Godot-Languages-Support/godot-lang-support#21]
https://github.com/dsrw/enu

Pretty wild, what should we do with this one?

**Comment by Vivraan** [comment]
Leave it be, it seems to be its own project - it even provides a custom Godot build to boot!
We could give it a mention in the Nim section or the Extra links section if you want, though.

**Comment by ShalokShalom** [comment]
Yes, some special category, its very limited in its use and very early in its development at all.

**Comment by Vivraan** [comment]
It's a 3D LOGO-like DSL using Godot and pragmagic's Nim bindings. Feels
more like something that'd go on the Nim Binding docs.

**Comment by ShalokShalom** [comment]
It uses Nimscript, afaik

**Comment by Vivraan** [comment]
TIL!

Would you like to consider it as a framework or a project that uses the Nim Lang Support? That's valid reason to have it in our doc.

Basically every language on our list could have links to projects which use them.

**Comment by ShalokShalom** [comment]
People who would look for it, probably dont care about the implementation, they look for something 3D oriented.
I could imagine, that we create an own miscellaneous section on the bottom and put it there. :)

**Comment by Vivraan** [comment]
That works too but we'd create redundancy in organising the content in the links section. Better alongside the language project itself.

**Comment by ShalokShalom** [comment]
Aaaand another curiosity, I think we could add an own category for them, on the complete bottom of the page, just to show some inspirational edge cases

https://github.com/ephread/inkgd

**Comment by Vivraan** [comment]
> Aaaand another curiosity, I think we could add an own category for them, on the complete bottom of the page, just to show some inspirational edge cases
> 
> https://github.com/ephread/inkgd

Seems similar to Escoria or other VN writing frameworks.

**Comment by ShalokShalom** [comment]
Maybe in its own category, called Miscellaneous or Curiosities.

**Comment by Vivraan** [comment]
> Maybe in its own category, called Miscellaneous or Curiosities.

sounds like something that should go to https://github.com/godotengine/awesome-godot.

**Comment by ShalokShalom** [comment]
Yeah, good idea 💡
